### ISSUE TITLE
Method 'get_Info' does not have an implementation.

### ISSUE THREAD

**Issue by kurt-o-matic** [bubibubi/EntityFrameworkCore.Jet#40]
When spinning up a little demo console app (.NET 4.8, EntityFrameworkCore 3.1.2, EntityFrameworkCore.Jet 2.2.0), and using the BloggingContext sample code here on the GitHub Wiki, the Add-Migration command results in the following error:
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.TypeLoadException: Method 'get_Info' in type 'EntityFrameworkCore.Jet.Infrastructure.Internal.JetOptionsExtension' from assembly 'EntityFrameworkCore.Jet, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' does not have an implementation.

   at System.Signature.GetSignature(Void* pCorSig, Int32 cCorSig, RuntimeFieldHandleInternal fieldHandle, IRuntimeMethodInfo methodHandle, RuntimeType declaringType)

   at System.Reflection.RuntimeMethodInfo.FetchNonReturnParameters()

   at System.Reflection.RuntimeMethodInfo.GetParameters()

   at System.Diagnostics.StackTrace.ToString(TraceFormat traceFormat)

   at System.Exception.GetStackTrace(Boolean needFileInfo)

   at System.Exception.ToString(Boolean needFileLineInfo, Boolean needMessage)

   at Microsoft.EntityFrameworkCore.Design.OperationExecutor.OperationBase.Execute(Action action)

   --- End of inner exception stack trace ---

   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)

   at System.Reflection.RuntimeConstructorInfo.Invoke(BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)

   at System.RuntimeType.CreateInstanceImpl(BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes, StackCrawlMark& stackMark)

   at System.Activator.CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, Object[] args, CultureInfo culture, Object[] activationAttributes)

   at System.Activator.CreateInstance(Type type, Object[] args)

   at System.Dynamic.UpdateDelegates.UpdateAndExecuteVoid4[T0,T1,T2,T3](CallSite site, T0 arg0, T1 arg1, T2 arg2, T3 arg3)

   at Microsoft.EntityFrameworkCore.Tools.OperationExecutorBase.InvokeOperationImpl(String operationName, IDictionary arguments)

   at Microsoft.EntityFrameworkCore.Tools.OperationExecutorBase.AddMigration(String name, String outputDir, String contextType)

   at Microsoft.EntityFrameworkCore.Tools.Commands.MigrationsAddCommand.Execute()

   at Microsoft.DotNet.Cli.CommandLine.CommandLineApplication.Execute(String[] args)

   at Microsoft.EntityFrameworkCore.Tools.Program.Main(String[] args)

Exception has been thrown by the target of an invocation.
```
I'm happy to fall back to earlier framework/library version or use another work-around to fix migration. Thanks for your efforts and attention!

**Comment by FreddyD-GH** [comment]
I believe this is trying to call an API that was added in EFCore 3.0.  EntityFrameworkCore.Jet currently only works with EFCore 2.2 for the moment.  @lauxjpn is currently working on upgrading the project to EFCore 3.1, but I don't believe he currently has an ETA for that.  To use EntityFrameworkCore.Jet in its current form, I would suggest downgrading the EntityFrameworkCore 3.1.2 package reference to version 2.2 and give that a try.

**Comment by lauxjpn** [comment]
@FreddyD-GH is right. This is the usual error you get when trying to use a EF Core 2.2 provider with EF Core 3.x.

I am about half way done with upgrading the provider code to 3.1.x and I will put in some hours over the weekend. We should be able to have a nightly build of this ready around the end of the upcoming week. (Though it will also depend a bit on the open `System.Data.OleDb` bugs or whether we can temporarily hack around them.)

**Comment by kurt-o-matic** [comment]
Thanks so much!

I took a guess and rolled back to EFC 2.2.6, since that looked like an
appropriate vintage. I then had to roll Microsoft.EntityFrameworkCore.Tools
back to 2.2.6, and then finally upgrade Microsoft.Extensions.Options up to
3.1.2, but at that point everything worked.

👍

Thanks for all your work on this project! After 25 years, you'd think I
would be done with MS Access, but every few years it haunts me once again.

Cheers!
Kurt
